includes/classes/blocks/class-template-parts.php

```
@@ -0,0 +1,248 @@
+			'label'       => __( 'Modals', 'tour-operator' ),
+			'description' => __( 'Template parts for customizing the modals.','tour-operator' ),
+			'icon'        => 'welcome-widgets-menus',
+			'area_tag'    => 'div',
+		];
+
+		if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+			error_log( 'Template Parts: Registered new areas via filter' );
+		}
+
+		return $areas;
+	}
```

PHPMD (2.15.0)

The method add_template_parts_area is not named in camelCase. (undefined)

(CamelCaseMethodName)